src/display/scale.js

```diff
@@ -0,0 +1,513 @@
+import { ImageKind } from "../shared/util.js";
+
+class ScaleJS {
+  isMobile() {
+    if (typeof navigator !== "undefined") {
+      return /Mobi/.test(navigator.userAgent);
```

Comment:
I'm not sure if this check is correct. For example, the document linked in issue #9474 loads just fine in Firefox Preview Nightly and Google Chrome on Android 10 (Samsung Galaxy Note 10+). Maybe a more specific user agent check could be introduced to avoid unnecessary scaling, which has a side effect of degrading image quality.

Reply:
We can use something like:  /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream; to narrow it down to IOS mobile devices.